src/fund_migration.rs

```diff
@@ -96,14 +110,37 @@ pub(crate) async fn migrate_funds(
     }
 }
 
-fn payout(_public_key: String, _invoice: String, _signature: String) -> Result<()> {
-    // TODO: Implement.
-    Ok(())
+fn fetch_legacy_balance(client: &Client, backend_url: &String, public_key: String) -> Result<u64> {
+    let variables = migration_balance::Variables {
+        node_pub_key: Some(public_key),
```

Comment:
Missing the hex string prefix (`\\x`)
```suggestion
        node_pub_key: Some(format!("\\xpublic_key")),
```

Reply:
This can actually be ignored. Just went over the service, we do not fully comply with hasuras hex encoding. Lets not send any prefixed hex strings for the migration

Reply:
Removed all prefixes in [3e2bf28](https://github.com/getlipa/lipa-lightning-lib/pull/650/commits/3e2bf286a007f49c9f637531a48c6b803f330262)